Modules/Quotas/test/Quotas.Domain.Tests/TierTests.cs

```diff
@@ -1,20 +1,36 @@
-﻿using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
 using FluentAssertions;
 using Xunit;
 
-namespace Backbone.Modules.Quotas.Domain.Tests
+namespace Backbone.Modules.Quotas.Domain.Tests;
+
+public class TierTests
 {
-    public class TierTests
+    [Fact]
+    public void Can_create_tier_with_valid_properties()
+    {
+        var tierId = "TIREYSCQI6XaMygco7Bw";
```

Comment:
You should always make it obvious that your test data are actually just test data.
In this example:
```suggestion
        var tierId = "TIRsomeTierId1111111";
```